### PR TITLE
incus: Add aliases completion

### PR DESCRIPTION
**Pull request by montag451**
This PR also improves the configuration loading because I didn't want to copy/paste on more time the code used to locate the user configuration file.

### CODE REVIEW

File: shared/cliconfig/file.go [hunk 1]
```diff
@@ -11,17 +13,57 @@ import (
...
+		return "", "", nil
+	}
+
+	configPath := os.ExpandEnv(path.Join(configDir, "config.yml"))
+
+	return configPath, filepath.Dir(configPath), nil
+}
+
 // LoadConfig reads the configuration from the config path; if the path does
 // not exist, it returns a default configuration.
```
Comment (stgraber) [inline shared/cliconfig/file.go:44]: That description needs updating as it no longer matches what the function does.

Comment (montag451) [reply shared/cliconfig/file.go:44]: Something like:
``` go
// LoadConfig reads the configuration from the config path; if the path does
// not exist, it returns a default configuration; if the given path is empty
// it tries to determine automatically the configuration file to load.
```

Comment (montag451) [reply shared/cliconfig/file.go:44]: Any advice?

Comment (stgraber) [reply shared/cliconfig/file.go:44]: Sorry, been traveling this week and struggling to keep up with Github :)
Yeah, that comment looks good to me.

Comment (montag451) [reply shared/cliconfig/file.go]: Done!

**Review by stgraber — APPROVED** [review]

**Comment by stgraber** [comment]
Thanks!

**Pull request merged**